kustomize/preprod/simple-app/deployment.yaml

```diff
@@ -13,6 +17,32 @@ spec:
       labels:
         app: simple-app
     spec:
+      volumes:
+        - name: env-volume
+          emptyDir: {}
+      initContainers:
+        - name: env-creator
+          image: alpine
+          command:
+            [
+              "/bin/sh",
+              "-c",
+              "echo MY_SECRET=$(echo $MY_SECRET) > /env/.env && echo NAME=$(echo $NAME) >> /env/.env",
```

Comment:
It will be annoying to add our secrets in the deployment, we could replace this by just calling `env` that will > into .env

Reply:
Updated! Please re-review and send me feedbacks. Thanks!